src/Workspaces/VisualBasic/Portable/Extensions/InvocationExpressionSyntaxExtensions.vb

```diff
@@ -47,7 +47,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
                 End If
             End If
 
-            Return True
+            Dim isOrdinaryMethod = semanticModel.GetSymbolInfo(invocationExpression.Expression).Symbol?.IsOrdinaryMethod
+            Return isOrdinaryMethod.HasValue AndAlso isOrdinaryMethod.Value
```

Comment:
```suggestion
            Return isOrdinaryMethod = True
```

Reply:
I tried that originally but got
BC30512	Option Strict On disallows implicit conversions from 'Boolean?' to 'Boolean'.
Shall I do `CBool(isOrdinaryMethod)`?

Reply:
oh.  interesting!  no.  just keep what you have here.

Reply:
#resolved